### ISSUE TITLE
Confusion matrix, ROC Curve Plot and Accuracy of model on Test set

### ISSUE THREAD

**Issue by singhsukhendra** [krishnaik06/Predicitng-Lungs-Disease-#1]
Can you please help if I want to evaluate the model accuracy on entire test set and how can we plot model 's confusion matrix , ROC Curve and observe precision, recall, sensitivity and specificity.

Thanks & regards

**Comment by Reshma-S13** [comment]
Yes Sir, help me also to generate the model's confusion matrix and ROC curve.

Thanks in Advance
